jest.config.cjs

```diff
@@ -2,6 +2,16 @@
 // For a detailed explanation regarding each configuration property, visit:
 // https://jestjs.io/docs/en/configuration.html
 
+// Any packages that are published only as ESM need to be listed here
+const externalNodeModules.join("|") = [
```

Comment:
👍 Nice, been trying to think of the best way to lay this out so it doesn't get so crowded.